### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by AndreyChernysh**
- [DEMO LINK](https://AndreyChernysh.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://AndreyChernysh.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,56 @@
...
+      <h2 class="product__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="product__code">
+        Product code: 195434
+      </p>
+
+      <div class="product__rating">
+        <div class="product__rating--stars stars">
```
Comment (referee1988) [inline src/index.html:32]: I think the product_rating block should be an element of the product_rating block. And accordingly have a class product__stars

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,56 @@
...
+          <div class="stars__star"></div>
```
Comment (referee1988) [inline src/index.html:33]: The stars__star element must have the class product__star

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,56 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="product__rating--rating-text rating-text">Reviews: 5</p>
```
Comment (referee1988) [inline src/index.html:39]: rating-text must be an element of the product block and have the class product_review

**Review by referee1988 — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (AndreyChernysh) [reply src/index.html:33]: I doubt it because stars__star is a dependent element of the stars block

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,56 @@
...
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0"
+    >
```
Comment (mykola-koval) [inline src/index.html:11]:
```suggestion
    <meta 
      name="viewport"
      content="width=device-width,
      user-scalable=no,
      initial-scale=1.0,
      maximum-scale=1.0,
      minimum-scale=1.0"
    >
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,133 @@
+/* Font Roboto 400 500 700*/
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+/* CSS */
```
Comment (mykola-koval) [inline src/style.css:4]: remove comments

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.product__img {
+  display: block;
```
Comment (mykola-koval) [inline src/style.css:35]: isn't it a default style for img tag?

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+  margin: 0 auto;
+  width: 160px;
+  height: 134px;
+}
+
+.product__name {
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  text-align: left;
```
Comment (mykola-koval) [inline src/style.css:45]: looks like a default style

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+.product__button {
+  background: #00acdc;
+  border: 1px solid #00acdc;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  transition: all 0.32s;
```
Comment (mykola-koval) [inline src/style.css:97]: do not add transition for `all`
add it for only what you need

**Review by mykola-koval — APPROVED** [review]
👍 gj

File: src/style.css (hunk 6 above)
Comment (AndreyChernysh) [reply src/style.css:35]: if I don't add this style, my tests don't pass because there is an empty space under the picture
